test/functional/grade_entry_forms_controller_test.rb

```diff
@@ -427,7 +432,8 @@ class GradeEntryFormsControllerTest < AuthenticatedControllerTest
                    :description => NEW_DESCRIPTION,
                    :message => NEW_MESSAGE,
                    :date => @grade_entry_form.date,
-                   :grade_entry_items_attributes => {'1' => @q1}}
+                   :grade_entry_items_attributes => {'1' => @q1},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.